### ISSUE TITLE
addMediaElement with mute:false and Microphone (from getUserMedia) conflict

### ISSUE THREAD

**Issue by blackbusters** [t-mullen/video-stream-merger#16]
Hi ! Thanks for this great library...

I've tried new addMediaElement features and it works well on playing from html video with sound (mute:false). But when i try to merge with camera and microphone from getUserMedia, it throws some error : 

```
Failed to execute 'createMediaElementSource' on 'BaseAudioContext': HTMLMediaElement already connected previously to a different MediaElementSourceNode.

at VideoStreamMerger.addMediaElement (video-stream-merger\src\index.js:130:46)
```

I believe this was caused by conflict using 2 BaseAudioContext from different source? is there any workaround if i want to merge and use this 2 audio from microphone and video at the same time?

**Comment by t-mullen** [comment]
I can't reproduce. Could you post a minimal example of what causes the issue, as well as the browser you're using?

**Comment by t-mullen** [comment]
```javascript
var merger = new VideoStreamMerger({
    width: 1000,
    height: 300,
    fps: 20
  })

merger.addMediaElement('myVideo', videoElement, {
    mute: false
  })
  merger.addStream(cameraMediaStream, {
    x: 0,
    y: 0,
    width: 100,
    height: 100
  })

  merger.start()
```

Merging a video file with a getusermedia stream works fine for me.

**Comment by blackbusters** [comment]
Sorry, my bad for misunderstood the situation..

It seems that the issue was caused by calling addMediaElement for more than one times, and before the next call, i use merger.removeStream(videoID) for hope to clean it up first, but maybe something is still there.

I'm using Chrome browser, and this example will raise an issue for me :

```
if (videoID != "") {
  merger.removeStream(videoID)
  videoID = "";
}

videoID = "testvid"
merger.addMediaElement(videoID , videoElement, {
	mute: false,
})

merger.start()
```

Basically, when my update button is pressed, it will try to merge all available input from user. The first try will have no issue, but on second time the button is pressed, it will trigger the error.

**Comment by t-mullen** [comment]
Oh I see the issue. I'll have a patch shortly.

**Comment by blackbusters** [comment]
Great ! 

thank you very much..

**Comment by t-mullen** [comment]
Fixed in `v2.8.4`. Thanks for the report 🙌
